### ISSUE TITLE
Config File Not Working

### ISSUE THREAD

**Issue by pjbrow** [akrennmair/newsbeuter#145]
Hi Guys

I can't get newsbeuter to use my config file.  I've installed newsbeuter 2.8 with brew on a mac running Mavericks.

I've encountered this while trying to configure to open links in a specific browser.

My config file called "config" is located in dir `/Users/Pat/.newsbeuter`.

The config command I've got in it is:

`browser w3m`

I've also tried a few other browsers, all with no joy.

The logs I get when I try and open with w3m are:

```
XDG: data directory '/Users/Pat/.local/share/newsbeuter' not accessible, using '/Users/Pat/.newsbeuter' instead.
Starting newsbeuter 2.8...
Loading configuration...done.
Opening cache...done.
Loading URLs from /Users/Pat/.dots/.newsbeuter/urls...done.
Loading articles from cache...done.
sh: lynx: command not found
sh: lynx: command not found
sh: lynx: command not found
sh: lynx: command not found
sh: lynx: command not found
Cleaning up cache...done.
```

So basically, it says that it has read the config file but it's not using it because it's still trying to use lynx as default browser.  I'm added `auto-reload yes` to config and that option doesn't seem to take either.

Anyone having a similar issue?

**Comment by yang-ling** [comment]
Try to use full path.
e.g. `browser /usr/bin/w3m`

I am on ArchLinux and this config works fine.

**Comment by h3xx** [comment]
If I gather you correctly, you have a "real" directory (not a symlink) called `.newsbeuter` in your home directory.

In it you have:
* a "real" file (not a symlink) called `config` with "browser w3m" in it
* a symbolic link `urls` pointing to `../.dots/.newsbeuter/urls`

**Comment by annndrey** [comment]
Try to set BROWSER variable in bash to w3m, it worked for me
export BROWSER=w3m

**Comment by Minoru** [comment]
@pjbrow, did any of the above suggestions help?

**Comment by jfm** [comment]
I know this is old, but I just ran into the same problem... The solution was, for me, to make sure that the last line ends with a newline.
